### PR TITLE
Rewrite comments of Localizable.strings to better localize

### PR DESCRIPTION
**Pull request by kkj1996**
### Platforms Impacted
- [ ✅] iOS
- [❌ ] macOS

### Description of changes

Rewrite comments of Localizable.strings to better localize, also pass the World-Readiness Validation.

### Verification

No need to test since it's only comments.

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

### Pull request checklist

This PR has considered:
- [✅ ] Light and Dark appearances
- [ ✅] iOS supported versions (all major versions greater than or equal current target deployment version)
- [ ✅] VoiceOver and Keyboard Accessibility
- [✅ ] Internationalization and Right to Left layouts
- [ ✅] Different resolutions (1x, 2x, 3x)
- [✅ ] Size classes and window sizes (iPhone vs iPad, notched devices, multitasking, different window sizes, etc)
- [✅ ] iPad [Pointer interaction](https://developer.apple.com/documentation/uikit/pointer_interactions)
- [ ✅] [SwiftUI](https://developer.apple.com/tutorials/swiftui) consumption (validation or new demo scenarios needed)
- [ ✅] Objective-C exposure (provide it only if needed)

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/fluentui-apple/pull/1355)

### CODE REVIEW

**Comment by kkj1996** [comment]
@microsoft-github-policy-service agree [company="Microsoft"]

**Comment by kkj1996** [comment]
@microsoft-github-policy-service agree
@microsoft-github-policy-service agree company="Microsoft"

**Comment by kkj1996** [comment]
@microsoft-github-policy-service agree company=Microsoft

File: ios/FluentUI/Resources/Localization/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -7,11 +7,22 @@
...
 "Accessibility.Alert" = "Alert";
+/* Accessibility dismiss label for common use */
 "Accessibility.Dismiss.Label" = "Dismiss";
+/* Accessibility dismiss hint for common use */
 "Accessibility.Dismiss.Hint" = "Double tap to dismiss";
+/* Accessibility done label for common use */
 "Accessibility.Done.Label" = "Done";
+/* Accessibility multi select hint for common use */
 "Accessibility.MultiSelect.Hint" = "Double tap to toggle selection";
+/* Accessibility select hint for common use */
```
Comment (harrieshin) [inline ios/FluentUI/Resources/Localization/en.lproj/Localizable.strings:20]: you are adding strings that didn't existed before?

Comment (kkj1996) [reply ios/FluentUI/Resources/Localization/en.lproj/Localizable.strings]: Fixed, it's a conflict issue.

**Review by harrieshin — APPROVED** [review]

**Pull request merged**